it/proxy_test.py

```diff
@@ -17,14 +17,15 @@
 
 import collections
 import os
-import shutil
-import tempfile
 
 import pytest
 
 import it
 from esrally.utils import process
 
+# pylint: disable=unused-import
+from it import fresh_log_file
+
```

Comment:
Do you really need this import for the fixture to be seen by pytest?

Reply:
Apparently, yes:
```
E       fixture 'fresh_log_file' not found
>       available fixtures: anyio_backend, anyio_backend_name, anyio_backend_options, benchmark, benchmark_weave, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, distribution_version, doctest_namespace, es_cluster, event_loop, httpserver, httpserver_listen_address, httpserver_ssl_context, make_httpserver, monkeypatch, pytestconfig, rally, record_property, record_testsuite_property, record_xml_attribute, recwarn, revision, setup_esrallyd, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory
>       use 'pytest --fixtures [testpath]' for help on them.
```